server/nginx/conf/nginx.conf

```diff
@@ -0,0 +1,92 @@
+worker_processes 1;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+    sendfile        on;
+    keepalive_timeout  65;
+
+    server {
+        server_name  auth.averycb.net;
```

Comment:
We'll need to change these to run locally, but we gotta demo so it's ok for now :)